qt/scientific_interfaces/ISISReflectometry/GUI/Batch/BatchPresenter.cpp

```diff
@@ -176,6 +178,11 @@ void BatchPresenter::resumeReduction() {
     notifyReductionPaused();
     return;
   }
+  if (m_jobRunner->getProcessAll() && m_mainPresenter->isProcessAllPrevented())
```

Comment:
I don't think it causes a problem but for completeness we should call `notifyReductionPaused` here as it does in the previous failure case to ensure we're in a paused state. Could the 3 `if` statements be combined so we only have to do this once?